mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipperTest.java

```diff
@@ -62,6 +73,15 @@
     private Zipper testee;
     private ByteArrayOutputStream output;
 
+    private static MessageResult MESSAGE_RESULT_1;
+    private static MessageResult MESSAGE_RESULT_2;
+
+    @BeforeAll
+    static void beforeAll() throws Exception {
+        MESSAGE_RESULT_1 = new MessageResultImpl(MESSAGE_1);
```

Comment:
That's sad a Message is needed to get a MessageResult... But there lmight be no way around for now....

Reply:
I don't see how to do it differently for now.

Reply:
Yes, mailbox-store dependency is OK in tests for now...